crowdsec-docs/docs/appsec/quickstart.md

```diff
@@ -51,26 +68,158 @@ You can now restart CrowdSec:
+
+1. Create a Remediation Component (Bouncer) API Key:
+
+```bash
+sudo cscli bouncers add test_waf -k this_is_a_bad_password
+API key for 'test_waf':
+
+   this_is_a_bad_password
+
+Please keep this key since you will not be able to retrieve it!
+```
+
+2. Emit a legitimate request to the AppSec Component:
+
+```bash
+curl -X POST localhost:7422/ -i -H 'x-crowdsec-appsec-uri: /test' -H 'x-crowdsec-appsec-ip: 42.42.42.42' -H 'x-crowdsec-appsec-host: foobar.com' -H 'x-crowdsec-appsec-verb: POST' -H 'x-crowdsec-appsec-api-key: this_is_a_bad_password'
+```
+
+Which will give us an answer such as:
+
+```bash
+HTTP/1.1 200 OK
+Date: Tue, 30 Jan 2024 15:43:50 GMT
+Content-Length: 36
+Content-Type: text/plain; charset=utf-8
+
+{"action":"allow","http_status":200}
+```
+
+3. Emit a malevolent request to the Appsec Component:
+
+:::info
+We're trying to access a `.env` file, a [common way to get access to some credentials forgotten by a developer.](https://app.crowdsec.net/hub/author/crowdsecurity/appsec-rules/vpatch-env-access)
+:::
+
+```bash
+curl -X POST localhost:7422/ -i -H 'x-crowdsec-appsec-uri: /.env' -H 'x-crowdsec-appsec-ip: 42.42.42.42' -H 'x-crowdsec-appsec-host: foobar.com' -H 'x-crowdsec-appsec-verb: POST' -H 'x-crowdsec-appsec-api-key: this_is_a_bad_password'
+
+```
+
+Our request is detected and blocked by the AppSec Component:
+
+```bash
+HTTP/1.1 403 Forbidden
+Date: Tue, 30 Jan 2024 15:57:08 GMT
+Content-Length: 34
+Content-Type: text/plain; charset=utf-8
+
+{"action":"ban","http_status":403}
+```
+
+Let's now delete our test API Key:
 
-Now that our Application Security Component is running in CrowdSec, we need to set up the remediation component to interact with it.
+```bash
+sudo cscli bouncers delete test_waf
+```
 
-Note that every remediation component may have different option names for this.
-I suggest you check directly in the remediation component documentation, but here we will use the [OpenResty remediation component](/u/bouncers/openresty) as an example.
+</details>
 
-To set up the AppSec in the OpenResty remediation component, we just need to set the Application Security Component URL previously exposed:
 
-```bash title="/etc/crowdsec/bouncers/crowdsec-openresty-bouncer.conf"
+## Remediation Component Setup
+
+With our AppSec Component active within CrowdSec, it's time to configure the remediation component to forward requests to it.
+
+:::info
+Different Remediation Components may offer various options for this setup, so consult the documentation for your specific component.
+:::
+
+To setup forwarding of requests in the Nginx remediation component, we'll modify its configuration file (`/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf`) by adding the following entries:
+
+```bash title="/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf"
 APPSEC_URL=http://127.0.0.1:7422
 # in case the AppSec run on the same machine, else provide the AppSec IP
```

Comment:
I believe comments are not allowed in the bouncer configuration, doesn't it ? If so, it would make sense to avoid showing some.